luarocks/src/luarocks/config.lua.win.in

```diff
@@ -12,8 +12,9 @@ rocks_trees = {
 }
 
 rocks_servers = {
-   [[https://raw.githubusercontent.com/torch/rocks/master]],
-   [[https://raw.githubusercontent.com/rocks-moonscript-org/moonrocks-mirror/master]]
+  [[https://raw.githubusercontent.com/diz-vara/rocks/master]],
```

Comment:
@hughperkins Not for official, certainly! I keep my repo in the list for some windows-specific changes (eg in luaffifb, imgraph).
It would be OK to remove it - but we should check every rock in the official repo